src/util/unicode.cpp

```diff
@@ -267,3 +267,22 @@ std::wstring utf8_to_utf16le(const std::string& in)
   std::wstring_convert<std::codecvt_utf8_utf16<wchar_t,0x10ffffUL,std::codecvt_mode::little_endian> > converter;
   return converter.from_bytes(in);
 }
```

Comment:
Unrelated to the code: the commit message should be cleaned, we don't really care about that conflict anymore.